template/script.sh.erb

```diff
@@ -5,19 +5,12 @@ modules_to_load, alternate_r_version = context.version.split(':')
 # Load the required environment
 setup_env () {
   module purge
-  # If you want to use Singularity it is recommended you setup a module similar
-  # to the included `example_module` and load it as:
-  #
-  #     module use "/path/to/example_module/modulefiles"
-  #     module load singularity rstudio_singularity/3.4.3
-  #
-  # Bonus points if you install the `example_module` on the system so you don't
-  # need to run `module use`.
-  module load <%= modules_to_load %>
-  <% if alternate_r_version %>
+  module use /users/PZS0002/mrodgers/.local/share/lmodfiles
+  module load <%= modules_to_load %> rserver-wrapper
```

Comment:
Maybe add a comment above this module load explaining that:

```
The rserver-wrapper module should set these environment variables:

    SINGULARITY_BINDPATH="/usr/local,/etc/profile.d/lmod.sh,/usr/share/lmod,/opt/intel,/opt/mvapich2,/usr/lib64"
    RSTUDIO_SERVER_IMAGE="/usr/local/project/ondemand/singularity/rstudio/rstudio_launcher_centos7.simg"
```

Or this should be added to the README as a dependency.

Actually it is also maybe worth specifying for the entire set_env() section what environment variables need to be set. Things that the rstudio and r modules set for us, along with rserver-wrapper. Because a site could copy this app, and just replace the body of setup_env with a set of "export VAR=VAL" statements.